### DISCUSSION TITLE
Ballance Incorrect?

### DISCUSSION THREAD

**Discussion by cmon69** [lightningnetwork/lnd#8591] [General]
I'm showing a ballance in my LND even after moving all my funds, why would this be?

How do I make sure this is accurate or resolve it?

![Untitled](https://github.com/lightningnetwork/lnd/assets/19421720/d0ffd1a3-7c74-46e4-b572-6d9968a15531)
![Untitled2](https://github.com/lightningnetwork/lnd/assets/19421720/18be8c3a-3d08-44a6-b301-24cb6814f8c0)

**Comment by bitromortac** [comment]
Could you give the output of `lncli walletbalance` (this can be done via `Lightning Shell` app on Umbrel, which may be need to be installed.) Did you open any channels?

**Reply by cmon69** [reply]
Here are the results from Lightning Shell.

lnshell@b6347dd5cc44:~$ lncli walletbalance
{
    "total_balance": "10147",
    "confirmed_balance": "0",
    "unconfirmed_balance": "10147",
    "account_balance": {
        "default": {
            "confirmed_balance": "0",
            "unconfirmed_balance": "10147"
        }
    }
}
lnshell@b6347dd5cc44:~$ 

No open channels.

**Reply by bitromortac** [reply]
Do you have any pending channels `lncli pendingchannels`, otherwise, try to look into `lncli listchaintxns` if you can spot the unconfirmed output. You may be able to sweep the remaining funds with the help of `lncli sendcoins --help`, using `--sweepall` and `--min_confs 0`.

**Reply by cmon69** [reply]
I doesnt show any pending channels

lnshell@b6347dd5cc44:~$ lncli pendingchannels
{
    "total_limbo_balance": "0",
    "pending_open_channels": [
    ],
    "pending_closing_channels": [
    ],
    "pending_force_closing_channels": [
    ],
    "waiting_close_channels": [
    ]
}
lnshell@b6347dd5cc44:~$

**Reply by cmon69** [reply]
Going thew the data that was produced after 'lncli listchaintxns' will take a little while.

**Reply by cmon69** [reply]
I went threw the output and did not find any 10147 which corresponds to the unconfirmed output. I wonder if it might be associated with a failed depost I tryed to make to Kraken a long time ago. I am in the process of contacting Kraken to get mote info.

**Reply by bitromortac** [reply]
It can be also an accounting problem in the wallet, restarting lnd with `--reset-wallet-transactions` could help here.

**Reply by cmon69** [reply]
Would you use the sweep before a reset?

**Reply by bitromortac** [reply]
This may be a utxo that doesn't exist (anymore), so `--reset-wallet-transactions` would scan your wallet for all the utxos. If the utxo is still present after the scan, you can sweep it.

**Reply by cmon69** [reply]
How would I do that, I tried doing `lncli sendcoins --sweepall --min_confs 0`: and got 

> [lncli] rpc error: code = Unknown desc = unable to broadcast sweep transaction: transaction rejected: output already spent

**Reply by bitromortac** [reply]
Could you check again if there's output from `lncli listunspent | grep -A3 -B2 10000`, we would need to find the UTXO lnd thinks exists. In order to rescan lnd you would either need to add this to your `lnd.conf` or to the lnd command line flags. As this is specific to Umbrel, I would ask in the Umbrel support channels how to do that.

**Comment by cmon69** [comment]
https://lightningshell.app sight down?
